src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentalPrice = 40;
+  const discountTreeDays = 20;
+  const discountSevenDays = 50;
+  const longPeriod = 7;
+  const shortPeriod = 3;
+
+  if (days < shortPeriod) {
+    return days * rentalPrice;
+  }
+
+  if (days >= shortPeriod && days < longPeriod) {
```

Comment:
There is no need to check 'days >= shortPeriod' because it is always true in this context due to the previous if statement. You can remove it.